### DISCUSSION TITLE
Missing JoinType on Include call

### DISCUSSION THREAD

**Discussion by TCBroad** [JasperFx/marten#2333] [Q&A] [answered]
Hi,

In the documentation for Marten, it says you can include a 3rd parameter on the Include method to specify the type of join performed. This 3rd parameter is not offered by intellisense and doesn't appear to be in the source code. Has it been removed with v5 and if so, how would you perform a LeftOuter join type? 
I've used this method in v3 and am trying to migrate to the latest and need an equivalent to this method/option.

Many thanks,

Tom

**Comment by jeremydmiller** [comment, marked as answer]
The mechanism for Include() is very different in V4/5 and the `JoinType` concept no longer exists.

**Reply by TCBroad** [reply]
Thanks for getting back to me. So assuming there isn't an equivalent method to do an outer join, are there any bits of Marten infrastructure I can use to roll my own? Quite a lot has changed since v3!

Cheers

**Reply by jeremydmiller** [reply]
The current behavior is in effect an "outer join" -- but there's no JOIN in the SQL statements.
